app/webpacker/components/Panel/pages/DuesRedirect/index.jsx

```diff
@@ -17,12 +17,17 @@ export default function DuesRedirect() {
   } = useLoadedData(wfcDuesRedirectsUrl);
   const xeroUsersFetch = useLoadedData(wfcXeroUsersUrl);
   const { save, saving } = useSaveAction();
-  const [open, setOpen] = React.useState(false);
-  const [toDeleteId, setToDeleteId] = React.useState();
-  const [formData, setFormData] = React.useState({ redirectType: 'Country' });
+  const [open, setOpen] = useState(false);
+  const [toDeleteId, setToDeleteId] = useState();
+  const [formData, setFormData] = useState({ redirectType: 'Country' });
 
   const handleFormChange = (_, { name, value }) => setFormData({ ...formData, [name]: value });
 
+  const handleRegionChange = (e, { value }) => handleFormChange(e, {
+    name: 'redirectFromCountryIso2',
+    value,
+  });
```

Comment:
Again, probably redundant now if you propagate the `name` to the country selector